ILSpy/NativeMethods.cs

```diff
@@ -193,6 +193,16 @@ public unsafe static string GetProcessNameFromWindow(IntPtr hWnd)
 				return null;
 			}
 		}
+
+		[DllImport("dwmapi.dll", PreserveSig = true)]
```

Comment:
Just a comment - too bad we can't yet use https://learn.microsoft.com/en-us/dotnet/standard/native-interop/pinvoke-source-generation